assets/src/edit-story/components/checklist/videoOptimizationCheckbox/index.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { TranslateWithMarkup, __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { trackClick } from '../../../../../../packages/tracking/src';
+import {
+  Checkbox,
+  Link,
+  Text,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
```

Comment:
Now that design system has move to it's own package, this will fail on merge. Please merge main.